package.json

```diff
@@ -26,11 +26,7 @@
     "lodash": "^4.17.11",
     "raven-js": "^3.27.0",
     "react": "16.5.2",
-<<<<<<< Updated upstream
-    "react-native": "~0.57.1",
-=======
-    "react-native": "https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz",
->>>>>>> Stashed changes
+    "react-native": "https://github.com/expo/react-native/archive/sdk-28.0.0.tar.gz",
```

Comment:
why we're using this version instead of 0.57 - that looks like a latest version?

Reply:
Oh..wait..seems like there is something wrong. Let me revert it again.